src/main/java/io/openliberty/tools/intellij/util/Constants.java

```diff
@@ -65,22 +65,19 @@ public final class Constants {
     public static final String VIEW_GRADLE_CONFIG_ACTION_ID = "io.openliberty.tools.intellij.actions.ViewGradleConfig";
     public static final String VIEW_EFFECTIVE_POM_ACTION_ID = "io.openliberty.tools.intellij.actions.ViewEffectivePom";
 
-    private static final HashMap<String, String> CORE_ACTIONS_MAP = new HashMap<String, String>() {
+    // action triggered from shift-shift "Search Everywhere" IntelliJ menu or "cmd/ctl + shift + A" Actions menu
+    public static final String GO_TO_ACTION_TRIGGERED = "GoToAction";
+
+    public static final LinkedHashMap<String, String> FULL_ACTIONS_MAP = new LinkedHashMap<String, String>() {
```

Comment:
Assuming this is meant to be read-only, you may want to wrap the Map using Collections.unmodifiableMap(). In general, mutable static objects can be a security issue if a 3rd party can access your class and overwrite your object with whatever they want.

Reply:
Good point, I'll add this thanks!